cookbooks/aws-parallelcluster-common/resources/nfs/nfs_centos7.rb

```diff
@@ -19,10 +19,19 @@
 unified_mode true
 
 use 'partial/_install_nfs4_and_disable'
+use 'partial/_configure'
 
 default_action :setup
 
 action :setup do
   action_install_nfs4
   action_disable_start_at_boot
 end
+
+action_class do
+  def override_server_template
+    edit_resource(:template, node['nfs']['config']['server_template']) do
+      cookbook 'nfs'
+    end
+  end
```

Comment:
Suggestion: can we avoid to repeat this code block putting it in a partial and import it when needed?

Reply:
This is the function used in the `:configure` action common to all OS that varies across operating systems.